.travis.yml

```diff
@@ -23,7 +23,7 @@ python:
 - 2.7
 script:
 - tox
-- coverage run --source pancloud -m py.test
+- coverage run --source pancloud -m py.test --ignore=examples
```

Comment:
Ignore `credentials_test.py` which is intermittently discovered and added to test suite by pytest